### PR TITLE
Refactor the Quiz 7 with tests

### PR DESCRIPTION
**Pull request by y-martinez**
## Summary
Close y-martinez/PyZombis#12.
There were functions names in English so they were translated.
The ids of exercises were changed to ``q7_#number``
The code was checked with ``flake8`` and ``black`` to resolve any problems of styling

## Checklist

- [X] Variables, function, and comments are translated into Spanish
- [X] Spell check done & typos fixed
- [X] All python code is PEP8 compliant
- [X] Test coverage with Playwright implemented
- [X] Reviewers assigned (all peers & at least 1 mentor)

### Test Quiz7_1 Good Result
test_quiz7_1_good_chromium

### Test Quiz7_1 Bad Result
test_quiz7_1_bad_chromium

### CODE REVIEW

File: _sources/quiz/Quiz7.rst [hunk 1]
```diff
@@ -7,445 +7,484 @@ Quiz - 7
...
    :nocodelens:
 
-   Desarrolle la función ``string_splosion`` que recibe una cadena ``s`` y devuelva una nueva cadena siguiendo el patron descrito en los ejemplos |br| |br|
+   Desarrolle la función ``expandir_cadena`` que recibe una cadena ``s`` y devuelva una nueva cadena siguiendo el patrón descrito en los ejemplos |br| |br|
    Ejemplos:|br|
-   ``string_splosion('Code')`` -> 'CCoCodCode' |br|
-   ``string_splosion('abc')`` -> 'aababc' |br|
-   ``string_splosion('ab')`` -> 'aab' |br|
+   ``expandir_cadena("Code")`` -> ``"CCoCodCode"`` |br|
+   ``expandir_cadena("abc")`` -> ``"aababc""` |br|
```
Comment (NicolasSandoval) [inline _sources/quiz/Quiz7.rst:62]: A new line is not displayed correctly

**Review by NicolasSandoval — CHANGES_REQUESTED** [review]
Great!

File: _sources/quiz/Quiz7.rst [hunk 2]
```diff
@@ -7,445 +7,484 @@ Quiz - 7
...
+Ejercicio 5
 -----------
 
-.. activecode:: hello_name
+.. activecode:: q7_5
    :nocodelens:
 
 
-   Desarrolle la función ``hello_name`` que recibe como parametro un cadena representa un nombre ``name`` y devuelva un saludo con este nombre |br| |br|
+   Desarrolle la función ``hola_usuario`` que recibe como parámetro un cadena representa un nombre ``nombre`` y devuelva un saludo con este nombre |br| |br|
```
Comment (LeoCumpli21) [inline _sources/quiz/Quiz7.rst:183]:
```suggestion
   Desarrolle la función ``hola_usuario`` que recibe como parámetro un cadena que representa un nombre ``nombre`` y devuelva un saludo con este nombre |br| |br|
```

File: _sources/quiz/Quiz7.rst [hunk 3]
```diff
@@ -7,445 +7,484 @@ Quiz - 7
...
+Ejercicio 11
+------------
 
-.. activecode:: remove_first_last
+.. activecode:: q7_11
    :nocodelens:
 
 
-   Desarrolle la función ``remove_first_last`` que recibe una cadena ``s`` de al menos dos caracteres y devuelva una cadena sin el primer y último carácter |br| |br|
+   Desarrolle la función ``remover_primer_ultimo`` que recibe una cadena ``s`` de al menos dos caracteres y devuelva una cadena sin el primer y último carácter |br| |br|
```
Comment (LeoCumpli21) [inline _sources/quiz/Quiz7.rst:377]:
```suggestion
   Desarrolle la función ``remover_primer_ultimo`` que recibe una cadena ``s`` de al menos dos caracteres y devuelva una cadena sin el primer ni el último caracter |br| |br|
```

**Review by LeoCumpli21 — APPROVED** [review]
LGTM :)
I left a few comments, but overall everything is fine.

**Review by angelasofiaremolinagutierrez — APPROVED** [review]
LGTM :)

**Review by NicolasSandoval — APPROVED** [review]

**Review by reingart — APPROVED** [review]
LGTM

**Pull request merged**